docs/user-guide/src/bmo/live_updates_servicing.md

```diff
@@ -0,0 +1,105 @@
+# Live updates (servicing)
+
+Live updates (servicing) enables baremetal-operator to conduct certain actions
+on already provisioned BareMetalHosts. These actions currently include:
+
+- [configuring firmware settings](./firmware_settings.md)
+- [updating BIOS and/or BMC firmware](./firmware_updates.md)
+
+Live updates (servicing) is an opt-in feature. Operators may enable this
+feature by creating a `HostUpdatePolicy` custom resource.
+
+## HostUpdatePolicy custom resource definition
+
+HostUpdatePolicy is the custom resource which controls applying live updates.
+Each part of the functionality can be controlled separately by setting the
+respective entry in the HostUpdatePolicy spec:
+
+- `firmwareSettings` - controls changes to firmware settings
+- `firmwareUpdates` - controls BIOS and BMC firmware updates
+
+### Allowed values for firmwareSettings and firmwareUpdates fields
+
+Each of the fields can be set to one of the two values:
+
+- `onReboot` - enables performing the requested change on next reboot, or
+- `onPreparing` - (default setting) limits applying this type of change to
+Preparing state (which only applies to nodes which are being provisioned)
+
+### Example HostUpdatePolicy definition
+
+Here is an example of a HostUpdatePolicy CRD:
+
+```yaml
+apiVersion: metal3.io/v1alpha1
+kind: HostUpdatePolicy
+metadata:
+  name: ostest-worker-0
+  namespace: openshift-machine-api
+spec:
+  firmwareSettings: onReboot
+  firmwareUpdates: onReboot
+```
+
+## How to perform Live updates on a BareMetalHost
+
+- create a HostUpdatePolicy resource with the name matching the BMH to be
+updated
+- use the format above, ensure `firmwareSettings` and/or `firmwareUpdates` is
+set to `onReboot`
+- make changes to [HostFirmwareSettings](./firmware_settings.md) and/or [HostFirmwareComponents](./firmware_updates.md) as required
+- make sure the modified resources are considered valid (see `Conditions`)
+- drain the node
```

Comment:
You need to add context: "if you're updating a Kubernetes node, make sure to drain it and mark as not schedulable"